bin/test/dune

```diff
@@ -1,8 +1,13 @@
 (executable
  (name        main)
- (public_name mdx-test)
+ (public_name ocaml-mdx-test)
  (package     mdx)
  (modes       byte)  ;; no native code toplevel libs yet
  (link_flags  -linkall)
  (preprocess (action (run %{bin:cppo} -V OCAML:%{ocaml_version} %{input-file})))
  (libraries   cli mdx.top))
+
+(install
+ (section bin)
+ (files
+  (main.exe as mdx-test)))
```

Comment:
I think it would be nice to not have both `mdx-test` and `ocaml-mdx-test`. Nobody is using `mdx-test` directly as far as I know.

Reply:
Done :)

Reply:
Bad commit name, see the second one :)